tests/kamereon/test_kamereon_vehicles.py

```diff
@@ -30,6 +30,17 @@
         "supports-hvac-status": False,
         "supports-location": True,
     },
+    "duster.1.json": {
+        "get_brand_label": "RENAULT",
+        "get_energy_code": "ESS",
+        "get_model_code": "XJD1SU",
+        "get_model_label": "NEW DUSTER",
+        "reports_charging_power_in_watts": False,
+        "uses_electricity": False,
+        "uses_fuel": True,
+        "supports-hvac-status": True,
+        "supports-location": True,
+    },
```

Comment:
@FRSTR can you please confirm that it is a fuel-only vehicle, and these specs are correct?

Reply:
Yes, this car uses petrol only. The fields look correct, though I'm not sure what get_energy_code stands for.